web/types/global.d.ts

```
@@ -0,0 +1,12 @@
+import * as React from 'react';
+
+declare global {
+  namespace JSX {
+    interface IntrinsicElements {
+      [elemName: string]: any;
+    }
+  }
+}
+
+// This export is needed to make this file a module
+export {};
```

Biome (1.9.4)

[error] This empty export is useless because there's another export or import.

This import makes useless the empty export.

Safe fix: Remove this useless empty export.


(lint/complexity/noUselessEmptyExport)